config/enabled.yml

```diff
@@ -545,6 +545,7 @@ Style/NumericLiterals:
                  Add underscores to large numeric literals to improve their
                  readability.
   StyleGuide: 'https://github.com/bbatsov/ruby-style-guide#underscores-in-numerics'
+  Autocorrect: false # This is unsound so isn't on by default
```

Comment:
I think that it would be a good idea to add some notes as to why this might not be a safe correction.